pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineClusterIntegrationTest.java

```diff
@@ -1307,7 +1308,8 @@ public void testCaseInsensitivity() {
 
     for (String query : queries) {
       try {
-        postQuery(query);
+        JsonNode response = postSqlQuery(query);
+        Assert.assertTrue(response.get("numSegmentsProcessed").asLong() >= 1L, query + " failed");
```

Comment:
You can directly use `assertTrue()`, same for other places

Reply:
Updated